cargo/Cargo.toml

```diff
@@ -25,6 +26,9 @@ prost-types = "*"
 rand = "*"
 simple_logger = "*"
 tokio = { version = "*", features = ["rt-core"] }
+# Using an old version that is supported by `cargo-raze`:
+# https://github.com/google/cargo-raze/issues/41#issuecomment-592274128
```

Comment:
Thanks for putting these comments in &ndash; they'll be useful breadcrumbs to help whoever next has to fight `cargo raze`.